packages/sdk-cli/src/models/localRelay/util.test.ts

```diff
@@ -0,0 +1,72 @@
+import { promises as fsPromises } from 'fs';
+import { tmpdir } from 'os';
+import { join } from 'path';
+import { RELAY_PID_FILE_NAME } from './const';
+import { isInt, readJsonFile } from './util';
+
+// Have to do this, because 'fsPromises' __must__ be mocked (which sets all fs methods to jest.fn())
+jest.mock('fs', () => {
+  const actualFs = jest.requireActual('fs');
+  const actualFsPromises = actualFs.promises as typeof fsPromises;
+
+  return {
+    ...actualFs,
+    promises: {
+      ...actualFsPromises,
+      readFile: jest.fn().mockImplementation(actualFsPromises.readFile),
+    },
+  };
+});
+
+describe('isInt', () => {
+  describe('false', () => {
+    it.each([undefined, null, Infinity, -Infinity])('%s', (x) =>
+      expect(isInt(x)).toBe(false),
+    );
+
+    it('number string', () => expect(isInt('5')).toBe(false));
+  });
+
+  describe('true', () => {
+    it('number', () => expect(isInt(5)).toBe(true));
+  });
+});
+
+describe('readJsonFile – actual file system', () => {
```

Comment:
Why the FS mocking above if you're keeping the actual FS versions of these tests?

Reply:
I...don't know. I didn't even mock it, it seems like I just wrapped the actual implementation around a `jest.fn()`. Removed